tests/test_part_image.py

```diff
@@ -88,7 +88,7 @@ def get_data_part_number(disk_image):
         return 4
 
 
-@pytest.mark.only_with_image('sdimg', 'uefiimg')
+@pytest.mark.only_with_image('img', 'uefiimg')
```

Comment:
I think this will not work with `meta-mender` tests for Yocto builds

Reply:
Hmm, I see. So add `img` as a separate entry?

Reply:
Or maybe I am wrong? Does yocto uses only uefi/bios/gpgt?

Reply:
Correct, this will not work with Yocto builds. We use the image type to distinguish between MBR and GPT images as well, although I *think* this is not used in mender-image-tests ATM. So adding it as a separate entry should probably work. If it doesn't, then the test will be silently skipped, so make sure it is actually run when fixing this.

Reply:
Alright, good to know. thx.